devsetup/scripts/standalone.sh

```diff
@@ -126,7 +122,7 @@ __EOF__
 
 export HOST_PRIMARY_RESOLV_CONF_ENTRY=${HOST_PRIMARY_RESOLV_CONF_ENTRY}
 export INTERFACE_MTU=${INTERFACE_MTU:-1500}
-export NTP_SERVER=${NTP_SERVER:-"clock.corp.redhat.com"}
+export NTP_SERVER=${NTP_SERVER:-"pool.ntp.org"}
```

Comment:
fair enough but this change has enough going on without also dealing with this default switch here

Reply:
could you elaborate? in rod jobs we provide an overriding interface https://github.com/29418/rdo-jobs/blob/6c8849c108a11ff1167ad9e726aeb5f039b9edfb/playbooks/data_plane_adoption/ci_framework_deploy_standalone_vm.yaml#L12 so I don't get the question

Reply:
you are changing the default. The change itself is probably right since we generally avoid exposing the interal urls in this space. It does mean though that all jobs previously overriding the default will now have to switch it. I am questioning if that is a necessary change as part of the series 
i.e. it is 'right' but is it needed here, on top of all the other changes being made.

Reply:
i just sanity checked that we are already setting this to d/stream servers for our dstream jobs so we should be OK on the default switch.
marking resolved.